k8s/spack/spackbot-spack-io/secrets-dummy.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Could you rename this to `secrets.yaml`?  That will match how the secrets are laid out in the other namespaces.

Reply:
we decided this should be called `secrets-dummy.yaml`, and that the real secrets files would be created from that.